packages/expo-localization/ios/LocalizationModule.swift

```diff
@@ -154,6 +155,20 @@ public class LocalizationModule: Module {
     sendEvent(CALENDAR_SETTINGS_CHANGED)
   }
 
+  static func getTemperatureUnit() -> String? {
+    let formatter = MeasurementFormatter()
+    formatter.locale = Locale.current
+
+    let temperature = Measurement(value: 0, unit: UnitTemperature.celsius)
+    let formatted = formatter.string(from: temperature)
+
+    guard let unitCharacter = formatted.last else {
+      return nil
+    }
+
+    return unitCharacter == "F" ? "fahrenheit" : "celsius"
```

Comment:
This should also work and is a bit more concise. I think we can assume that if, for some reason, the unit is not present in the formatted string, it should be `celsius`.

```suggestion
    return formatter.last == "F" ? "fahrenheit" : "celsius"
```